src/lib/components/skip-nav.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+  import Link from '$lib/holocene/link.svelte';
+  import { translate } from '$lib/i18n/translate';
+</script>
+
+<Link
+  id="skip-nav"
+  href="#content"
+  class="absolute left-1/2 translate-y-[-100%] focus:translate-y-0 z-[100] p-1"
```

Comment:
Why not `translate-y-full`?

Are we okay with the bespoke z-index?

Reply:
> Why not translate-y-full?

Needs to be a negative (we want it off the page).

> Are we okay with the bespoke z-index?
We can update this to `z-50`, but then we'll need to update `NavContainer` to something else (and potential a few other things). I was 50/50 and so went with bespoke.

Reply:
Oh, I can't read. Why not `-translate-y-full`?

Reply:
Yea, I think maybe we should just update to standardized z-indices.

Reply:
> Why not -translate-y-full?

TIL

> update to standardized z-indices.

Opinions on what these might be? Or perhaps I should just add 100 to the config for now...